src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    protected double radius;
```

Comment:
Change the 'protected' access modifier for the 'radius' variable to 'private' to follow encapsulation principles.